src/model.ts

```diff
@@ -0,0 +1,6 @@
+export interface Model {
```

Comment:
Whoops, forgot about this.. will delete it unless you all think this is a good idea.